cgmanifest.json

```diff
@@ -50,6 +50,16 @@
       },
       "DevelopmentDependency": false
     },
+    {
+      "Component": {
+        "Type": "git",
+        "git": {
+          "RepositoryUrl": "https://github.com/intel/tinycbor",
+          "CommitHash": "755f9ef932f9830a63a712fd2ac971d838b131f1"
+        }
+      },
+      "DevelopmentDependency": false
```

Comment:
This needs to be true. It isn't a real customer facing dependency of the SDK (like curl), just a devdep.

Reply:
oh i forgot to change that. Thanks.